publish-token/pom.xml

```diff
@@ -4,17 +4,17 @@
 
 	<modelVersion>4.0.0</modelVersion>
 
+    <groupId>fi.thl.covid19</groupId>
```

Comment:
different indentation?

Reply:
Good question. It seems to be fine in my end.